wp-content/themes/cpipr/lcdm-single.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Template Name: Lcdm single
+ * Template Post Type: post
+ * Description: Shows the post with formatting for custom layout display?
+ */
+
+    // Load up our meta data and whatnot
+    the_post();
+?>
+
+<!DOCTYPE html>
+    <!--[if lt IE 7]> <html <?php language_attributes(); ?> class="no-js ie6"> <![endif]-->
+    <!--[if IE 7]>    <html <?php language_attributes(); ?> class="no-js ie7"> <![endif]-->
+    <!--[if IE 8]>    <html <?php language_attributes(); ?> class="no-js ie8"> <![endif]-->
+    <!--[if IE 9]>    <html <?php language_attributes(); ?> class="no-js ie9"> <![endif]-->
+    <!--[if (gt IE 9)|!(IE)]><!--> <html <?php language_attributes(); ?> class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="<?php bloginfo( 'charset' ); ?>" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+        <?php
+        /**
+         * The template for displaying the header
+         *
+         * Contains the HEAD content and opening of the id=page and id=main DIV elements.
+         *
+         * @package Largo
+         * @since 0.1
+         */
+        ?>
+        <title>
+            <?php
+                global $page, $paged;
+                wp_title( '|', true, 'right' );
+                bloginfo( 'name' ); // Add the blog name.
+
+                // Add the blog description for the home/front page.
+                $site_description = get_bloginfo( 'description', 'display' );
+                if ( $site_description && ( is_home() || is_front_page() ) )
+                    echo " | $site_description";
+
+                // Add a page number if necessary:
+                if ( $paged >= 2 || $page >= 2 )
+                    echo ' | ' . 'Page ' . max( $paged, $page );
+            ?>
+        </title>
+        <link rel="profile" href="http://gmpg.org/xfn/11" />
+        <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
+        <?php
+            if ( is_singular() && get_option( 'thread_comments' ) )
+            wp_enqueue_script( 'comment-reply' );
+            wp_head();
+        ?>
+
+        <!-- ScrollTo -->
+        <script src="<?php echo get_stylesheet_directory_uri(). '/lib/scrollTo/jquery.scrollTo.min.js' ?>" type="text/javascript"></script>
+
+        <!-- Simple Parallax -->
+        <script src="<?php echo get_stylesheet_directory_uri(). '/lib/simpleParallax/dist/simpleParallax.min.js' ?>" type="text/javascript"></script>
+
+    </head>
+
+    <body <?php body_class(); ?>>
```

Comment:
@jmoreira-linkux Again, is there a specific reason you're hardcoding the `body` opening here? It might be better to just use the `largo_top` [action hook](https://largo.readthedocs.io/developers/hooksfilters.html?highlight=largo_top#template-hooks) to do whatever you need to do here as opposed to copying over the HTML.